packages/backend/src/index.ts

```diff
@@ -16,3 +16,15 @@ export { defineStorage } from '@aws-amplify/backend-storage';
 
 // function
 export { defineFunction } from '@aws-amplify/backend-function';
+
+// re-export trigger types as "<category><trigger>Handler"
+export type {
+  CustomMessageTriggerHandler as AuthCustomMessageHandler,
+  PreAuthenticationTriggerHandler as AuthPreAuthenticationHandler,
+  PreSignUpTriggerHandler as AuthPreSignUpHandler,
+  PostAuthenticationTriggerHandler as AuthPostAuthenticationHandler,
+  PostConfirmationTriggerHandler as AuthPostConfirmationHandler,
+  UserMigrationTriggerHandler as AuthUserMigrationHandler,
+  S3Handler as StorageHandler,
+  AppSyncResolverHandler as DataResolverHandler,
+} from 'aws-lambda';
```

Comment:
This is going to create quite a noise when somebody
`import * as backend from amplify-backend`
and then tries to `backend.` in IDE.

Should we
1. Namespace this in `amplify-backend/functions/types` or so?
2. Move this to `backend-function` and export only there ?